src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/AnswerSteps.java

```diff
@@ -153,6 +153,11 @@ public void answer_text_is(String text) {
         answerRequest.setAnswerText(text);
     }
 
+    @Given("^the answer state is (.*)$")
+    public void answer_state_is(String state) {
+        answerRequest.setAnswerState(state);
+    }
+
```

Comment:
This is just a small change because I did like the style of theAnswerStateIsSUBMITTED below